<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Infection Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_LC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Constante de Proporcionalidade (k):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leProporcionalityConstant">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>441</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_AC">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de Pessoas (n):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Qualidade:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leQtdPeoples">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>741</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>1241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCustomPlot" name="qcPlotGraph" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>1221</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="hsResolution">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>107</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Processar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbRealTime">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>50</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tempo Real</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>100</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar em PNG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClean">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>70</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>110</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tempo Máximo:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbMaxTime">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>100</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbResistance">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Simular Resistência</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Constante Alfa:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbAlphaConst">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>50</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
    <property name="singleStep">
     <number>1000</number>
    </property>
   </widget>
   <zorder>label_LC</zorder>
   <zorder>leProporcionalityConstant</zorder>
   <zorder>label_AC</zorder>
   <zorder>label</zorder>
   <zorder>leQtdPeoples</zorder>
   <zorder>qcPlotGraph</zorder>
   <zorder>hsResolution</zorder>
   <zorder>line_2</zorder>
   <zorder>line</zorder>
   <zorder>checkBox</zorder>
   <zorder>cbRealTime</zorder>
   <zorder>pushButton</zorder>
   <zorder>pbClean</zorder>
   <zorder>label_2</zorder>
   <zorder>sbMaxTime</zorder>
   <zorder>cbResistance</zorder>
   <zorder>label_3</zorder>
   <zorder>sbAlphaConst</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1238</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfection_Simulator">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionAjuda"/>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuInfection_Simulator"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAjuda">
   <property name="text">
    <string>Ajuda</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar em PNG</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
